azure-pipelines.yml

```diff
@@ -722,6 +722,16 @@ steps:
       } else {
         Write-Host "The folder does not exist most likely because the devices hasn't been built."
       }
```

Comment:
These changes here in the YAML must have been unwillingly added.

Reply:
Yes, I'm going to have another PR, just wanted to test this.